cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ScoreLangConstants.java

```diff
@@ -44,6 +44,15 @@
     String LOOP_KEY = "loop";
     String BREAK_LOOP_KEY = "breakOn";
 
+    // async loop
+    String ASYNC_LOOP_KEY = "async_loop";
+    String ASYNC_LOOP_STATEMENT_KEY = "asyncLoopStatement";
+    String TASK_AGGREGATE_KEY = "taskAggregateValues";
+    String BRANCHES_CONTEXT_KEY = "branches_context";
+    String SPLIT_ITEM_KEY = "splitItem";
```

Comment:
only keys that are shared between compiler and runtime supposed to be in this constants class.